platform/working-with-vsp/policies-work-norms/sensitive-guidance.md

```diff
@@ -50,6 +50,10 @@ Areas like the VA's network toplogy, especially regarding the TIC and surroundin
 
 Personal contact information for individuals working on VA.gov should generally not be public.
 
+### Internal links
+
+Internal links can be included in publicly facing documentation and must be labeled with `[Internal - Requires VA Account]` so that public users are informed that internal links won't work for them. 
```

Comment:
```suggestion
Internal links can be included in publicly facing documentation and must be labeled with `[Internal]` so that public users are informed that internal links won't work for them. 
```

I think that's going to be pretty wordy, particularly for inline links within text --- @KevinMHoffmanUSDS are you OK with this shorter equivalent?

Reply:
@gunsch @ElijahLynn The information architect in me says that "[Internal]" isn't sufficiently specific. How about we change it to "[Login required]"?

Reply:
@KevinMHoffmanUSDS Hmm, the pedant in me doesn't think that's accurate. Sometimes it might require SOCKS access, sometimes it might require CAG access, sometimes it might be a link to the sensitive GitHub repo, sometimes it might be a system that requires a separate login.

Can you share more about what's not specific enough about "[Internal]"? Would "[Private]" be a useful alternative?

Reply:
I like "[Internal]" because it is easy to remember and if we do anything more than that, I suspect it won't be followed precisely.

Reply:
Hey there, @ElijahLynn and @gunsch - apologies for my delay in response. "[Internal]" works, and thanks for engaging in a discussion on this issue!